Support/Scripts/prepare-android-ndk.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+import sys
+import argparse
+from typing import Any
+import urllib.request
+import zipfile
+import os
+import shutil
+
+try:
+    import progress.bar
+except ImportError:
+    print("python3 -m pip install progress to see progress bars")
+
+bar = None
+bar_message = None
+
+
+def update_progress_bar(chunk_number: int, maximum_chunks: int, total_size: int):
+    global bar
+    if "progress.bar" not in sys.modules:
+        return
+    if bar is None:
+        bar = progress.bar.Bar(bar_message, max=total_size / maximum_chunks)
+
+    bar.next()
```

Comment:
Is this going to update the bar according to the arguments `update_progress_bar()` receives? Doesn't seem like it to me.